client/src/apis/chessGameApi.js

```diff
@@ -1,13 +1,23 @@
 import axios from 'axios';
 import Config from '../config/globalConfig';
 
+const handleErrorResponse = err => ({ isError: true, error: err });
+
 const getRating = ({ playerOneRating, playerTwoRating, winner }) => {
   const url = `${Config.serverUrl}/get-rating?playerOneRating=${playerOneRating}&playerTwoRating=${playerTwoRating}&winner=${winner}`;
   return axios.get(url)
```

Comment:
why not post request?

Reply:
We are not updating any rating in the database, only calculating new ratings on server.
once we have the schema for "games" we might change this to socket event or post request.